implementations/c/ockam/codec/tests/payload_aead_aes_gcm_test.c

```diff
@@ -23,19 +23,19 @@ int _test_codec_payload_aead_aes_gcm_setup(void** state)
 
   aag_test_payload = malloc(CODEC_MAX_VLU2_SIZE);
   if (NULL == aag_test_payload) {
-    status = OCKAM_ERROR_INTERFACE_CODEC;
+    status = -1;
```

Comment:
Convert `-1` to a macro or const for readability.

Reply:
See answer above